.run/bsh-ui.sh

```
@@ -0,0 +1,65 @@
+COLOUR="\033[35m" # Set a colour for highlighting the output
+LINE_END="\033[0m\n" # End of line and colour reset
+
+REPO_NAME="business-support-hub-front-end"
+RESOURCE_API_ENDPOINT="http://localhost:8080/site/bsh-api/resourceapi"
+
+# Default: rebuild
+rebuild=true
+
+# Update to the latest version of the main branch
+if [ -d "../${REPO_NAME}" ]; then
+  printf  "${COLOUR}Updating the local repository... ${LINE_END}"
```

Shellcheck (0.11.0)

[info] Don't use variables in the printf format string. Use printf '..%s..' "$foo".

(SC2059)